src/controllers/publish.ts

```diff
@@ -639,17 +639,54 @@ export const fetchDimensionPreview = async (req: Request, res: Response, next: N
                     };
                     res.redirect(req.buildUrl(`/publish/${dataset.id}/lookup/${dimension.id}`, req.language));
                     return;
-                case 'age':
-                case 'ethnicity':
-                case 'geography':
-                case 'religion':
-                case 'sexGender':
+                case 'Geog':
+                case 'Age':
+                case 'Eth':
+                case 'Gen':
+                case 'Rlgn':
```

Comment:
These feel like they should be enums eventually.

Reply:
I agree this probably needs to be an enum soon... but I think we'll leave it and see where the chips fall on Reference Data.